server/superdesk/media/image_tests.py

```diff
@@ -24,7 +24,49 @@ def test_extract_meta_json_serialization(self):
         with open(self.img, mode='rb') as f:
             meta = get_meta(f)
 
-        self.assertEquals(meta['ExifImageWidth'], 32)
-        self.assertEquals(meta['ExifImageHeight'], 21)
-        self.assertEquals(meta['Make'], 'Canon')
-        self.assertEquals(meta['Model'], 'Canon EOS 60D')
+        self.assertEqual(meta['ExifImageWidth'], 32)
+        self.assertEqual(meta['ExifImageHeight'], 21)
+        self.assertEqual(meta['Make'], 'Canon')
+        self.assertEqual(meta['Model'], 'Canon EOS 60D')
+
+
+class ExifMetaWithGPSInfoExtractionTestCase(TestCase):
+
+    fixtures = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'fixtures')
+    img = os.path.join(fixtures, 'iphone_gpsinfo_exif.JPG')
+
+    def test_extract_meta_json_serialization(self):
+        expected_gpsinfo = {
+            'GPSImgDirection': (14794, 475),
+            'GPSLongitudeRef': 'W',
+            'GPSImgDirectionRef': 'T',
+            'GPSLongitude': ((70, 1), (38, 1), (3946, 100)),
+            'GPSAltitudeRef': 0,
+            'GPSLatitudeRef': 'S',
+            'GPSAltitude': (105587, 183),
+            'GPSLatitude': ((33, 1), (26, 1), (1150, 100)),
+            'GPSTimeStamp': ((19, 1), (59, 1), (5117, 100))
+        }
+
+        with open(self.img, mode='rb') as f:
+            meta = get_meta(f)
+
+        self.assertEqual(meta.get('ExifImageWidth', None), 400)
+        self.assertEqual(meta.get('ExifImageHeight', None), 300)
+        self.assertEqual(meta.get('Make', None), 'Apple')
+        self.assertEqual(meta.get('Model', None), 'iPhone 5')
+        self.assertEqual(meta.get('GPSInfo', None), expected_gpsinfo)
+
+
+class ExifMetaExtractionUserCommentRemovedTestCase(TestCase):
+
+    fixtures = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'fixtures')
+    # this image has UserComment exif data value as 'test'
+    img = os.path.join(fixtures, 'iphone_gpsinfo_exif.JPG')
+
+    def test_extract_meta_json_serialization(self):
+
+        with open(self.img, mode='rb') as f:
+            meta = get_meta(f)
+
+        self.assertIsNone(meta.get('UserComment', None))
```

Comment:
If the second argument to `get()` is omitted, it already defaults to `None` - but it's true that the form above is more explicit, which is often not a bad thing.

Nevertheless, if you don't care about the case when `meta['UserComment'] == None` and are actually checking if `UserComment` exists at all, an even more Pythonic way to do it is:
```py
self.assertNotIn('UserComment', meta)
```
(I think that's the case here, since you are popping this key from the dictionary in `media/image.py`.